src/main/java/com/hillel/items_exchange/dto/ChildDto.java

```diff
@@ -19,7 +22,8 @@
     @PositiveOrZero(message = "{invalid.id}")
     private long id;
     @NotNull(message = "{invalid.not-null}")
-    private String sex;
+    @Enumerated(EnumType.STRING)
+    private ChildGender sex;
```

Comment:
You should add Swagger annotation to describe allowed values of this Enum

Reply:
Thanks, I will study it.

Reply:
I see swagger shows field type and allowable values without any annotations. E.g. see AdvertisementDto.dealType